docs/_examples/integration-tutorial-code/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "_integration-tutorial-code",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "node example-eslint-integration.test.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "eslint": "^8.39.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.27.5"
```

Comment:
Do we need these in the example project?

Reply:
removing. 

i had these and the .eslintrc file b/c i was using it to lint the code i was writing